diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/MultipleModifiersSequence.kt

```diff
@@ -46,6 +51,15 @@ class MultipleModifiersSequence(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    private fun isSamInterfaces(parent: ASTNode, node: ASTNode): Boolean {
```

Comment:
I think we also need to check, that this interface contains only one abstract method, isn't it?

Reply:
I think that for this we need to make a separate rule, this no longer concerns the logic of the order of modifiers.

Reply:
Fixed